tests/longevity/results/1.0.0.md

```diff
@@ -0,0 +1,234 @@
+{"level":"info","ts":"2023-09-27T17:19:27Z","logger":"statusUpdater","msg":"Skipping updating Nginx Gateway status because not leader"}
+{"level":"info","ts":"2023-09-27T19:54:13Z","logger":"statusUpdater","msg":"Skipping updating Gateway API status because not leader"}
+{"level":"info","ts":"2023-09-27T19:54:15Z","logger":"statusUpdater","msg":"Skipping updating Gateway API status because not leader"}
+{"level":"info","ts":"2023-09-27T19:54:24Z","logger":"statusUpdater","msg":"Skipping updating Gateway API status because not leader"}
+{"level":"info","ts":"2023-09-27T19:54:25Z","logger":"statusUpdater","msg":"Skipping updating Gateway API status because not leader"}
+```
+
+There are two problems:
+
+- The NGF pod lost its leadership, even though no other NGF pods were
+  running -- https://github.com/nginxinc/nginx-gateway-fabric/issues/1100
+- The leader elector logs are not structural -- https://github.com/nginxinc/nginx-gateway-fabric/issues/1101
+
+### NGINX
+
+#### Error Log
+
+Errors:
+
+```text
+resource.type="k8s_container"
+resource.labels.pod_name="nginx-gateway-b6cdb65cd-h8bgs"
+resource.labels.namespace_name="nginx-gateway"
+resource.labels.container_name="nginx"
+severity=ERROR
+SEARCH("`[warn]`") OR SEARCH("`[error]`")
+```
+
+No entries found.
+
+#### Access Log
+
+Non-200 response codes in NGINX access logs:
+
+```text
+severity=INFO
+"GET" "HTTP/1.1" -"200"
+```
+
+No such responses.
+
+## Key Metrics
+
+### Containers memory
+
+![memory.png](1.0.0-images/memory.png)
```

Comment:
We've been using full paths in our docs instead of relative ones.

Reply:
I see many examples where we don't do that, starting from the readme.

If the comment specifically about images, I think absolute links make sense here https://github.com/nginxinc/nginx-gateway-fabric/tree/main/docs because the images in a dedicated folder. But for cases when images are in the same or subfolder of a current doc, I think such a rule is overkill. And we don't always follow it -- https://github.com/nginxinc/nginx-gateway-fabric/blob/main/design/control-data-plane-separation/design.md